Default autoComplete to off on BpkAutosuggest input

Andy-Rennie opened the pull request.

matthewdavidson commented:
Cheers @Andy-Rennie ! Can we have a test (snapshot is fine) and an entry in the UNRELEASED.md please 🙏

Andy-Rennie commented:
Added tests and updated UNRELEASED, noticed that snapshot was already there with autoComplete="off" maybe defaulting in tests for some reason? So added a test to check the value is defined and can be overridden
